ruleset/rules/0755-office365_rules.xml

```diff
@@ -0,0 +1,928 @@
+        <if_sid>91532</if_sid>
+        <field name="office_365.RecordType" type="osregex">^33$</field>
+        <description>Office 365: Events related to DLP classification in SharePoint.</description>
+        <options>no_full_log</options>
+        <group>ComplianceDLPSharePointClassification</group>
+    </rule>
+
+    <rule id="91562" level="5">
+        <if_sid>91532</if_sid>
+        <field name="office_365.RecordType" type="osregex">^34$</field>
+        <description>Office 365: Campaign-related events from Microsoft Defender for Office 365.</description>
+        <options>no_full_log</options>
+        <group>ThreatFinder</group>
+    </rule>
+
+    <rule id="91563" level="3">
+        <if_sid>91532</if_sid>
+        <field name="office_365.RecordType" type="osregex">^35$</field>
+        <description>Office 365: Microsoft Project events.</description>
+        <options>no_full_log</options>
+        <group>Project</group>
+    </rule>
+
+    <rule id="91564" level="3">
+        <if_sid>91532</if_sid>
+        <field name="office_365.RecordType" type="osregex">^36$</field>
+        <description>Office 365: SharePoint List events.</description>
+        <options>no_full_log</options>
+        <group>SharePointListOperation</group>
+    </rule>
+
+    <rule id="91565" level="3">
+        <if_sid>91532</if_sid>
+        <field name="office_365.RecordType" type="osregex">^37$</field>
+        <description>Office 365: SharePoint comment events.</description>
+        <options>no_full_log</options>
+        <group>SharePointCommentOperation</group>
+    </rule>
+
+    <rule id="91566" level="3">
+        <if_sid>91532</if_sid>
+        <field name="office_365.RecordType" type="osregex">^38$</field>
+        <description>Office 365: Events related to retention policies and retention labels in the Security and Compliance Center.</description>
+        <options>no_full_log</options>
+        <group>DataGovernance</group>
+    </rule>
+
+    <rule id="91567" level="3">
+        <if_sid>91532</if_sid>
+        <field name="office_365.RecordType" type="osregex">^39$</field>
+        <description>Office 365: Kaizala events.</description>
+        <options>no_full_log</options>
+        <group>Kaizala</group>
+    </rule>
+
+    <rule id="91568" level="5">
+        <if_sid>91532</if_sid>
+        <field name="office_365.RecordType" type="osregex">^40$</field>
+        <description>Office 365: Security and compliance alert signals.</description>
+        <options>no_full_log</options>
+        <group>SecurityComplianceAlerts</group>
+    </rule>
+
+    <rule id="91569" level="5">
+        <if_sid>91532</if_sid>
+        <field name="office_365.RecordType" type="osregex">^41$</field>
+        <description>Office 365: Safe links time-of-block and block override events from Microsoft Defender for Office 365.</description>
+        <options>no_full_log</options>
+        <group>ThreatIntelligenceUrl</group>
+    </rule>
+
+    <rule id="91570" level="3">
```

Comment:
Security and Compliance Center events, could it be level 5?

Reply:
Done